erpnext/accounts/doctype/journal_entry/journal_entry.py

```diff
@@ -406,6 +406,8 @@ def set_exchange_rate(self):
 		for d in self.get("accounts"):
 			if d.account_currency == self.company_currency:
 				d.exchange_rate = 1
+			elif d.exchange_rate < 1:
```

Comment:
Erm just realized that the exchange rate being < 1 doesn't mean that it's valid, it could also be > 1. It's just that in our use case (CAD > USD) USD more valuable. If you reversed those, we'd end up with a rate > 1

Reply:
Maybe just a != then?

Reply:
Or just `elif not d.exchange_rate`?

Reply:
Note - Check the default value, if 1 then handle, otherwise if it's None then the above should work.